test/integration/targets/vmware_host_sriov/tasks/main.yml

```diff
@@ -0,0 +1,102 @@
+# Test code for the  module vmware_host_sriov
+
+- when: vcsim is not defined
```

Comment:
This block will skip all tests in integration test. @goneri please can you confirm if this block is required?

Reply:
The question is more: Can we run the integration test-suite with govcsim? if it's the case, we should drop the line. Otherwise, the integration tests will still be run, but only with a regular lab.

Reply:
as I understood there are no possibility to test it with `govcsim`
```
$ curl -sk https://user:pass@127.0.0.1:8989/about | grep -i configManager  
FAIL: 1
```
May be I am wrong,  please point me.  

> if it's the case, we should drop the line. therwise, the integration tests will still be run, but only with a regular lab.

I thought, that `- when: vcsim is not defined` - instructs not to use `vcsim` and forward directly to regular lab tests. It's not correct?